CONTRIBUTING.md

```diff
@@ -92,10 +95,22 @@ Infrastructure engineers may want to look at the
 [`infrastructure` module's `README`](./infrastructure/README.md), which includes
 how to configure your local machine or a virtual machine as a video bridge host.
 
+### 2.2.1 Developing Mailers.
+
+Set [SMTP\_\* environment variables to configure Action Mailer](.env.example).
+
+To preview mailers, visit http://localhost:3000/rails/mailers/
+
+### 2.2.2 Debugger
+
+To run `pry` or `byebug`, run `bin/connect web`.
```

Comment:
It would be nice if we linked to a great guide on debugging Rails applications; but I don't know what that guide would be. Perhaps @KellyAH or @anaulin know of one?

Reply:
I found the official guides for each (well, the pry one is a quickstart and a video) and put them in here.

Reply:
Thank you!!!